pom.xml

```diff
@@ -41,10 +38,10 @@
   </developers>
 
   <scm>
-    <connection>scm:git:ssh://github.com:jenkinsci/${project.artifactId}-plugin.git</connection>
-    <developerConnection>scm:git:ssh://git@github.com/jenkinsci/${project.artifactId}-plugin.git</developerConnection>
+    <connection>scm:git:https://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
+    <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
```

Comment:
GitHub deprecated ssh over git access some time ago and we will ban it from the plugin pom soon.